packages/ckeditor5-build-classic/webpack.config.js

```diff
@@ -72,6 +72,15 @@ module.exports = {
 							injectType: 'singletonStyleTag',
 							attributes: {
 								'data-cke': true
+							},
+							insert: function addToStyles( styleElement ) {
```

Comment:
I'd add a comment here do describe why the function is here and what it's doing.